fem/fe_coll.hpp

```diff
@@ -177,15 +217,23 @@ class H1_FECollection : public FiniteElementCollection
    { return H1_dof[GeomType]; }
    virtual const int *DofOrderForOrientation(Geometry::Type GeomType,
                                              int Or) const;
+
    virtual const char *Name() const { return h1_name; }
    virtual int GetContType() const { return CONTINUOUS; }
+   int GetBasisType() const { return b_type; }
+
    FiniteElementCollection *GetTraceCollection() const;
 
-   int GetBasisType() const { return b_type; }
    /// Get the Cartesian to local H1 dof map
    const int *GetDofMap(Geometry::Type GeomType) const;
+   /// Variable order version of GetDofMap
+   const int *GetDofMap(Geometry::Type GeomType, int p) const;
```

Comment:
The way `GetDofMap` is implemented will change once #975 is merged (which should be very soon). I believe that won't affect this PR, but would be good to double-check.